website/src/components/DashboardHeader/DashboardHeader.jsx

```diff
@@ -44,15 +52,14 @@ class DashboardHeader extends PureComponent {
       },
     ];
     if (teacher) {
-      Icons.unshift({
-        type: 'add_circle',
-        onClick: () => { window.location = '/create/class'; },
-        key: 'add_circle',
-      });
+      Icons.unshift(...TeacherIcons);
     }
     return (
       <Header className="Header">
-        <ProfileInfo name={firebase.auth().currentUser ? firebase.auth().currentUser.displayName : 'User Name'} email={firebase.auth().currentUser ? firebase.auth().currentUser.email : 'email@domain.com'} />
+        <ProfileInfo
+          name={firebase.auth().currentUser ? firebase.auth().currentUser.displayName : 'User Name'}
```

Comment:
I know this wasnt you but we can probably put this into a variable `let currentUser = firebase.auth().currentUser` and then do this `currentUser ? currentUser.displayName ... `

Reply:
aii ye